common/config/params.go

```diff
@@ -393,6 +396,10 @@ type Params struct {
 	// statistics error.
 	VoteStatisticsHeight uint32
 
+	// RegisterCRByDIDHeight defines the height to support register and update
+	// CR by CID and DID.
+	RegisterCRByDIDHeight uint32
```

Comment:
Will the name be changed to RegisterCRByCIDHeight?

Reply:
Starting from this height would support registering CR by did, so I think RegisterCRByDIDHeight is better.